back/engines/commercial/texting/db/migrate/20220308181809_create_texting_campaigns.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTextingCampaigns < ActiveRecord::Migration[6.1]
+  def change
+    create_table :texting_campaigns, id: :uuid do |t|
+      t.string :phone_numbers, array: true, default: [], null: false
```

Comment:
I didn't know it was possible to specify a string type as array. What's the advantage compared to json? Could it be inspected within an SQL query (for example, find all campaigns that include a specific number)?

Reply:
> What's the advantage compared to json?

You can write an object in json. You cannot do it with array.

> Could it be inspected within an SQL query (for example, find all campaigns that include a specific number)?

Yes, sure https://stackoverflow.com/questions/39643454/postgres-check-if-array-field-contains-value